EIPS/eip-2666.md

```diff
@@ -0,0 +1,136 @@
+---
+eip: 2666
+title: Repricing of precompiles and Keccak256 function
+author: Alex Vlasov (@shamatar)
+discussions-to: https://ethereum-magicians.org/t/eip2666-global-precompiles-repricing-and-many-more-discussion-thread/4332
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-05-22
+requires: 1352, 2046, 2565
+---
+
+## Simple Summary
+This EIP tries to fix the problems with Ethereum precompiles and built-in EVM function:
+- EIP-2046 changes a `STATICCALL` cost to precompile and it may be necessary to adjust costs of some precompiles that have taken old large cost (`700` gas) into account 
+- Some precompiles are overpriced and their pricing formulas to not reflect the structure of underlying function
+- Keccak256 built-in function in EVM has pricing that does not reflect underlying hash function structure
+- Better specify behavior of how gas handled if the precompile call fails (DDoS handling behavior)
```

Comment:
```suggestion
Change gas price of `BNADD`, `BNMUL`, `SHA256`, `RIPEMD` and `Keccak256`.
```
Simple summary should be very terse, one or two sentences at most, and should cut straight to the heart of this specification with no dilly dallying on the "why" or the "how".